app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RecipesController < ApplicationController
+  def index
+    if params[:query].present?
+      @recipes = RecipeFacade.search(params[:query])
+    else
+      @recipes = []
```

Comment:
This isn't really good error handling IMO. We should take what the server gives us back as a response.